object/slicer/slicer.go

```diff
@@ -289,15 +312,16 @@ type PayloadWriter struct {
 	currentEpoch uint64
 	sessionToken *session.Object
 
-	buf bytes.Buffer
+	payloadBuffer      []byte
+	extraPayloadBuffer []byte
```

Comment:
Why a whole additional buffer?

Reply:
`payloadBuffer` provided by user can be small (e.g. 1K). If so, and data exceeds external buffer, we need an extra one to produce child objects as efficient as possible from the system POV, i.e. with payload size = limit (except for the tail element)

anyway, `len(payloadBuffer) + len(extraPayloadBuffer) <= min(payloadSize, payloadLimit)`